packages/lambda-powertools-middleware-correlation-ids/package.json

```diff
@@ -9,8 +9,9 @@
   "license": "MIT",
   "author": "Yan Cui",
   "dependencies": {
-    "@dazn/lambda-powertools-correlation-ids": "^1.8.2",
-    "@dazn/lambda-powertools-logger": "^1.9.0"
+    "@dazn/lambda-powertools-correlation-ids": "^1.5.0",
+    "@dazn/lambda-powertools-logger": "^1.8.0",
+    "aws-sdk": "^2.496.0"
```

Comment:
Can make this a dev dep as it'll be available in the lambda runtime.

Reply:
Generally shouldn't depend on the version of AWS SDK in the runtime as it introduces subtle differences between what you tested and what's running in Lambda. The version in the Lambda runtime can also change without any action from you, so you lose immutable infra as well.